addons/block_code/ui/main_panel.gd

```diff
@@ -82,6 +84,14 @@ func _on_delete_dialog_confirmed(block_code_node: BlockCode):
 	undo_redo.commit_action()
 
 
+func _try_migration():
+	var version: int = _current_block_code_node.block_script.version
+	if version == Constants.CURRENT_DATA_VERSION:
+		# No migration needed.
+		return
+	push_warning("Migration not implemented from %d to %d" % [version, Constants.CURRENT_DATA_VERSION])
```

Comment:
I think this is good enough for now, but in the future we might want to beef this up. If the stored data version is less than the current data version, migrate it. If the stored data version is greater than the current data version, then someone has already migrated the data to a newer plugin version and is now trying to run an older version of the plugin. Since the older plugin version by definition doesn't know about the newer data format, I think the only thing to do there would be to refuse to load the script and bring up some kind of error message.